src/App.js

```diff
@@ -21,21 +21,22 @@ function App() {
     <ThemeProvider theme={theme}>
     <Container>
       <BrowserRouter>
-      <Nav themeToggle={toggleTheme} />
-      <Link to="/">Home</Link>
-      <SubNav />
+      <NavigationList toggleTheme={toggleTheme} />
       <Routes>
-      <Route exact path="/" element="">Home
-      </Route>
+      <Route exact path="/" element="">Home</Route>
       <Route path="/Portfolio" element="">Portfolio</Route>
-      <Route path="/coins">Coins</Route>
       <Route path="/coin/:coinid" element={<Coin />}></Route>
-    </Routes>
-      <Charts />
-    <div className="CoinTab">
-        <CoinsList />
-      </div>
+    </Routes> 
       </BrowserRouter>
+      <div className="OrganizeCharts">
+        <div className="ChartWrapper">
+        <VolumeChart />
+        </div>
+        <div className="ChartWrapper">
+        <LineChart />
+        </div>
+      </div>
+      <CoinsList />
```

Comment:
this should in the forst route, no?

Reply:
what do you mean? Like the "Home" route?

Reply:
When I do that, it errs